src/pages/docs/invoke/making-custom-apis.mdx

```diff
@@ -105,8 +105,8 @@ The result of the Rib expression is whatever the result of the last line/express
 The API definitions can be uploaded to Golem using one of the following methods:
 
 - Use the [Golem CLI](/docs/cli) tool
-- Use the [Rest APIs](rest-api)
-- Use [Golem Console](console) (Only available in Golem Cloud)
+- Use the [Rest APIs](/docs/rest-api)
+- Use [Golem Console](https://console.golem.cloud/) (Only available in Golem Cloud)
```

Comment:
We wanted to have a console subpage it is just not written yet (See #55 - cc @nicoburniske )

Reply:
sure, i know, just kinda it seem to me to have link (which working) there for now, then something which fail, after we will have page we can update those links